docs/recipes/ImplementingUndoHistory.md

```diff
@@ -0,0 +1,181 @@
+# Implementing Undo History
+
+To add undo functionality (or any other functionality) to your existing
+reducers, you need to create a higher-order reducer, which is a function
```

Comment:
It's alright :) I don't mind writing more - I'll try to work on it tomorrow, but I can't promise anything, right now are very busy times for me. Don't worry, I have it noted and will work on it as soon as I find time for it! :grin: